non-pinned.json5

```diff
@@ -1,11 +1,11 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    ":combinePatchMinorReleases",
-    ":dependencyDashboard",
-    ":separateMajorReleases",
+    ":combinePatchMinorReleases", // Do not separate patch and minor upgrades into separate PRs for the same dependency
+    ":dependencyDashboard", // Enable the dependency dashboard
+    ":separateMajorReleases", // Major releases will never be combined with minor/patch, so we can hold off on major and get incremental minors
     "config:base",
```

Comment:
@Hinton I can't find any documentation of `config:base` as an option. Do you have a link to what that does for us?

Reply:
They renamed it to recommended apparently.